### DISCUSSION TITLE
Decoding error

### DISCUSSION THREAD

**Discussion by sagraplus** [upscayl/upscayl#738] [Q&A]
I sure hope someone can help. I am getting the "Decoding Error". I am pulling my hair out after researching on the site (FAQ, etc) and on Google trying to solve this on my own. The only thing I found was something about "removing the GPU ID". I don't know what that is and cannot find information on the. Also I am not sure I understand folder permissions, but I can tell you I user many different programs (music, graphics, OpenOffice) and have never had a problem like this. I hope some kind person will help me out. This should be easy. Thanks in advance for help.

![Screenshot_55](https://github.com/upscayl/upscayl/assets/10904492/eec7ee5e-d78c-456e-a804-1e5b351601dc)

**Comment by sagraplus** [comment]
I should add, I have a Windows 10 Desktop, Intel Quad core, 16 gig ram.

**Comment by aaronliu0130** [comment]
> Yes, unfortunately. NCNN Vulkan requires a Vulkan-compatible GPU. Upscayl won't work with most iGPUs or CPUs.

A workaround exists for Windows and Linux in #390.

If you had followed the pinned bug report guide, you'd have seen instructions to read that.
